.travis.yml

```diff
@@ -1,16 +1,13 @@
 language: ruby
 
+script: make test
+
 rvm:
-  - 1.8.7
-  - 1.9.3
-  - 2.0
   - 2.1
   - 2.2
-  - 2.3.0
-  - jruby-18mode
-  - jruby-19mode
-  - jruby-9.0.5.0
-  - rbx-2
+  - 2.3.3
+  - jruby-9
+  - rbx-3
```

Comment:
Should we drop rbx-3 testing/support as well? We allow failures anyway and usage of it is probably low anyway